src/vue/backend/src/store/common/store/mutationTypes.js

```diff
@@ -0,0 +1,5 @@
+export const SET_ITEM = 'SET_ITEM'
```

Comment:
The path `store/common/store` is kinda funky, not not keep it up a level? Same with mutations.js